discord/guild.py

```diff
@@ -758,6 +758,8 @@ def _create_channel(self, name, overwrites, channel_type, category=None, **optio
         ----------
         name: str
             The new name of the guild.
+        description: str
+            The new description of the guild.
```

Comment:
I feel like this needs slightly more detail. What is a description? Where is it in the GUI?

If this is an attribute we can receive, then the documentation could be on the attribute.